core/common/src/main/java/alluxio/conf/Configuration.java

```diff
@@ -641,4 +644,54 @@ private static Optional<Properties> loadProperties(InputStream stream) {
     }
     return Optional.of(properties);
   }
+
+  private static GetConfigHashPResponse getConfigHash(InetSocketAddress address,
+      AlluxioConfiguration conf)
+      throws AlluxioStatusException {
+    GrpcChannel channel = null;
+    try {
+      LOG.debug("Alluxio client (version {}) is trying to get configHash from meta master {}",
+          RuntimeConstants.VERSION, address);
+      channel = GrpcChannelBuilder.newBuilder(GrpcServerAddress.create(address), conf)
+          .disableAuthentication().build();
+      MetaMasterConfigurationServiceGrpc.MetaMasterConfigurationServiceBlockingStub client =
+          MetaMasterConfigurationServiceGrpc.newBlockingStub(channel);
+      GetConfigHashPResponse response = client.getConfigHash(
+          GetConfigHashPOptions.newBuilder().build());
+      LOG.debug("Alluxio client has get configHash from meta master {}", address);
+      return response;
+    } catch (io.grpc.StatusRuntimeException e) {
+      throw new UnavailableException(String.format(
+          "Failed to handshake with master %s to get cluster configHash values: %s",
+          address, e.getMessage()), e);
+    } catch (UnauthenticatedException e) {
+      throw new RuntimeException(String.format(
+          "Received authentication exception during get configHash connect with host:%s", address),
+          e);
+    } finally {
+      if (channel != null) {
+        channel.shutdown();
+      }
```

Comment:
is this copied from `Configuration.loadConfiguration()`? Can you extract the common part so we don't copy the code?